mixins/localize-mixin.js

```diff
@@ -143,10 +152,26 @@ export const LocalizeMixin = superclass => class extends superclass {
 		return Array.from(langs);
 	}
 
+	static _getAllLocalizeResources(possibleLanguages) {
+		let resourcesLoadedPromises;
+		const superCtor = Object.getPrototypeOf(this);
+		if ('_getAllLocalizeResources' in superCtor) {
+			resourcesLoadedPromises = superCtor._getAllLocalizeResources(possibleLanguages);
+		} else {
+			resourcesLoadedPromises = [];
+		}
+		// eslint-disable-next-line no-prototype-builtins
+		if (this.hasOwnProperty('getLocalizeResources') || this.hasOwnProperty('resources')) {
```

Comment:
Could this be an `&&` instead of an `||`? Wondering if the way it's written, a component could have a `resources` property but not be using localization at all but it would now get picked up by this code. A long shot, but it'd be nice to confirm that they have `resources` AND they're actually using the localize mixin.

Reply:
This is written as an `||` because the `_hasResources` checks that `getLocalizeResources` exists somewhere in the inheritance chain. If it doesn't exist then this code is skipped altogether.

The `||` is needed because when using `LocalizeStaticMixin`, `getLocalizeResources` is defined in the superclass while `resources` is defined in the subclass so `this.hasOwnProperty('getLocalizeResources')` is only `true` for the superclass. Since the `LocalizeStaticMixin` defines an empty `resources` block calling `getLocalizeResources` from the `superclass` causes the actual terms to be missed.